Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -1003,8 +1008,10 @@ def import_ioc_without_approval(client: Client, file_id, classification, confide
         severity=severity,
         tags=tags,
         trustedcircles=trustedcircles
-    )})
-
+    )
+    for key in meta_from_params.copy():
+        meta.update({key: meta_from_params[key]})
```

Comment:
Three opinionated alternatives: 
1. instead of creating `meta_from_params`, can we just do `meta |= assign_params(...)`?
2. instead of these two rows, `meta |= meta_from_params`
3. replace 1014 with `ioc_to_import.update({"meta": meta | meta_from_params})`